### DISCUSSION TITLE
Description of all the binaries in the release archive

### DISCUSSION THREAD

**Discussion by santosh** [firecracker-microvm/firecracker#3111] [Q&A] [answered]
There are currently 4 binaries in the releases archive which is published on the releases page. 

- firecracker
- jailer
- seccompiler-bin
- rebase-snap

I understand that jailer is a production ready version of firecracker binary. What about other two?

**Comment by dianpopa** [comment, marked as answer]
Hi @santosh,

You can find information on both the other binaries (rebase-snap and seccompiler-bin) in the latest [CHANGELOG](https://github.com/firecracker-microvm/firecracker/blob/1285e4a356c61c7a8455dee4273df77bd05d0c16/CHANGELOG.md).
